### ISSUE TITLE
Registration CODE FAIL

### ISSUE THREAD

**Issue by carjr** [tgalal/yowsup#2844]
Before I was not even sending the SMS, now I was able to send the SMS but he failed to verify.

yowsup-cli registration -R XXX -XXX  -c c.config
yowsup-cli  v3.2.0
yowsup      v3.2.3

Copyright (c) 2012-2019 Tarek Galal
http://www.openwhatsapp.org

This software is provided free of charge. Copying and redistribution is
encouraged.

If you appreciate this software and you would like to support future
development please consider donating:
http://openwhatsapp.org/yowsup/donate

I 2019-08-03 10:58:21,227 yowsup.common.http.warequest - {"login":"551798213XXXX","status":"f       ail","reason":"temporarily_unavailable","retry_after":3600}

{
    "__version__": 1,
    "cc": "55",
    "client_static_keypair": "8BSewlGkI/4CaHuJSPR7GIEdezlSq2rA/2GTpqzBBG5w0o72G7zeU65x3axecuz       FeQjvnsUHWBe8/eyNLnNjDg==",
    "expid": "7TQ6lfMhSAquWd1NGPOm9g==",
    "fdid": "f9f53e68-1eef-4a02-b2f2-0319714bdfa7",
    "id": "3d92YhpneMeXqO7SenFVV+Tt6v8=",
    "mcc": "000",
    "mnc": "000",
    "phone": "551798213XXXX",
    "sim_mcc": "000",
    "sim_mnc": "000"
}
status: fail
retry_after: 3600
login: 551798213XXXX

and even after waiting for the seconds request, it fails and already tried to change the phone number and also comes back with this error

**Comment by bryanjhv** [comment]
Have you even read the README which clearly says that you will get banned if registering a number with Yowsup? I guess you didn't, so congrats with your freshly-blocked number.

**Comment by carjr** [comment]
Yes, I read so I said that before I could not even send the SMS for confirmation but now sent.

**Comment by Nottt** [comment]
So is not possible to use this project atm?

**Comment by nosuchip** [comment]
@Nottt yes, exactly.

**Comment by carjr** [comment]
I was able to register a number and use yowsup normally!

**Comment by ShiSHcat** [comment]
Same with latest version, i tried with 2 numbers and it worked fine, status ok

**Comment by Nottt** [comment]
I tried to register with a number from the app 2ndline, and it failed with this:

```
status: fail
reason: old_version
login: 16XXXXXXXX
```

And I installed with `pip install yowsup`

**Comment by AKwoKWH** [comment]
same, getting the the "old_version" error.
I am using the latest version.
Anyone have the same issue?

my code is:
```
yowsup-cli registration --requestcode sms --config-phone xxxxxxxxxxx --config-cc 65 --config-mcc 406 --config-mnc 03
```

```
I 2019-10-13 13:16:43,966 yowsup.common.http.warequest - b'{"login":"xxxxxxxxxxx","status":"fail","reason":"old_version"}\n
```
